pkg/horizon/secret.go

```diff
@@ -0,0 +1,33 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package horizon
+
+import (
+	"encoding/base64"
+
+	"math/rand"
+	"time"
+)
+
+// GenerateSecret -
+func GenerateSecret() string {
+	rand.Seed(time.Now().UnixNano())
+	data := make([]byte, 10)
+	for i := 0; i < 10; i++ {
+		data[i] = byte(rand.Intn(10))
+	}
+	return base64.StdEncoding.EncodeToString(data)
```

Comment:
Does it make sense to base64 encode it here when any string we push into the secret will be b64 encoded anyway? For example, the current blahsecret:

```
❯ oc get secret -o jsonpath='{.data}' horizon | jq
{
  "horizon-secret": "YmxhaFNlY3JldA=="
}
```

Reply:
I picked up this implementation from keystone operator (the code was used to generate fernet keys) but I noticed we don't need base64 encode here. will make this more simple.